101-recovery-services-weekly-backup-policy-create/azuredeploy.json

```diff
@@ -32,6 +32,14 @@
         "description": "Any Valid timezone, for example:UTC, Pacific Standard Time. Refer: https://msdn.microsoft.com/en-us/library/gg154758.aspx"
       }
     },
+    "instantRpRetentionRangeInDays": {
+      "type": "int",
+      "defaultValue": 5,
+      "allowedValues": [5],
```

Comment:
why is this a param if it only allows on value?

Reply:
Fixed this. Removed this parameter and passed directly value